### PR TITLE
Task4: Integration with Database

### PR DESCRIPTION
**Pull request by AmberCode**
All mandatory and optional tasks are done.

**BE:** 
get products - [ https://w0z0xq2ju8.execute-api.eu-central-1.amazonaws.com/dev/products](https://w0z0xq2ju8.execute-api.eu-central-1.amazonaws.com/dev/products). 
productById - [ https://w0z0xq2ju8.execute-api.eu-central-1.amazonaws.com/dev/products/7567ec4b-b10c-48c5-9345-fc73c48a80a0]( https://w0z0xq2ju8.execute-api.eu-central-1.amazonaws.com/dev/products/7567ec4b-b10c-48c5-9345-fc73c48a80a0). 
Create product (POST) https://w0z0xq2ju8.execute-api.eu-central-1.amazonaws.com/dev/products
Payload schema:
```
{
      type: "object",
      properties: {
        title: { type: 'string', minLength: 1 },
        description: { type: 'string' },
        price: { type: 'integer', minimum: 0 },
        count: { type: 'integer', minimum: 0 }
      },
      required: ["title", "price", "count"],
}
```

sample:

```
{
    "title": "x2",
    "description": "x2",
    "price": 2,
    "count": 2
}

```
**FE:**
[https://d2cvlb8s8ekb12.cloudfront.net](https://d2cvlb8s8ekb12.cloudfront.net)

### CODE REVIEW

**Comment by VladislavLobakh** [comment]
Wrong response code (https://w0z0xq2ju8.execute-api.eu-central-1.amazonaws.com/dev/products/aaa), should be 404

**Comment by VladislavLobakh** [comment]
Screenshot 2022-07-27 at 20 40 09
403 Error (spelling error, should be products)

**Comment by VladislavLobakh** [comment]
https://w0z0xq2ju8.execute-api.eu-central-1.amazonaws.com/dev/product
also spelling error, not working from FE app

File: product-service/src/db/task4script.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+create table stock (
+	product_id uuid not null primary key,
+	count integer,
+	foreign key (product_id) references product(id)
+);
+
+
+INSERT INTO public.product
+(id, title, description, price)
+VALUES('7567ec4b-b10c-48c5-9345-fc73c48a80aa', 'product title1', 'This product1', 1);
```
Comment (dmytro-shershnev1) [inline product-service/src/db/task4script.sql:19]: @AmberCode Why do you insert `id` explicitly in `product` table if you already have `uuid_generate_v4()` function? You can omit it.

**Pull request merged**